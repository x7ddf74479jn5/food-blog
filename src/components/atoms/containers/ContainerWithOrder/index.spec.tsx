import renderer from "react-test-renderer";

import { ContainerWithOrder } from ".";

describe("components/atoms/containers/ContainerWithOrder", () => {
  it("snapshot", () => {
    const tree = renderer.create(<ContainerWithOrder order="order-1">children</ContainerWithOrder>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
