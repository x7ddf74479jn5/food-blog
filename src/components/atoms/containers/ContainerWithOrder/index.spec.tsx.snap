// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/containers/ContainerWithOrder snapshot 1`] = `
<div
  className="flex flex-col order-1 items-center w-full space-y-8 "
>
  children
   
</div>
`;
